src/stories/form.stories.tsx

```diff
@@ -339,6 +341,62 @@ stories.add('Tags selector', withState({ value: ['Manchester', 'Utrecht'] })
     )
 );
 
+stories.add('Search field', withState({ value: null })
+    (withInfo('Search field')
+        (({ store }) =>
+            <Tile>
+                <div style={{ width: 350 }}>
+                    <SearchField
+                        id="search_field"
+                        value={store.state.value}
+                        onChangeHandler={(id, value) => store.set({ value })}
+                        label={text('Label', 'Search')}
+                        labelLayoutWidth={selectV2('Label Layout Width', widthFractionOptions, widthFractionDefaultValue)}
+                        labelWidthBreakpoint={selectV2('Label Layout Breakpoint', breakpointOptions, breakpointDefaultValue)}
+                        placeholder={text('Placeholder', 'Search')}
+                        validationError={text('Validation error message', '')}
+                        focusOnLoad={boolean('Focus on load', true)}
+                        staticField={boolean('Static field', false)}
```

Comment:
I think we should remove the fields from storybook that don't make sense, like `staticField` and `required`.